app/models/lifehack.rb

```diff
@@ -4,6 +4,43 @@ class Lifehack < ActiveRecord::Base
   validates :title, presence: true
   before_validation :strip_whitespace
 
+  def short_description
+    desired_length = 40
+    unless ( description.nil? || description.strip.empty? )
```

Comment:
Don't use parentheses around the condition of an `unless`.<br>Space inside parentheses detected.